helix-term/src/application.rs

```diff
@@ -161,15 +161,20 @@ impl Application {
             editor.open(&path, Action::VerticalSplit)?;
             // Unset path to prevent accidentally saving to the original tutor file.
             doc_mut!(editor).set_path(None);
-        } else if !args.files.is_empty() {
-            if args.open_cwd {
-                // NOTE: The working directory is already set to args.files[0] in main()
-                editor.new_file(Action::VerticalSplit);
-                let picker = ui::file_picker(".".into(), &config.load().editor);
+        } else if let Some((first, _)) = args.files.first() {
+            let mut skip_files = 0;
```

Comment:
It should be a little cleaner here to make an iterator over the files so we don't need to keep track of indices:

```rust
let files = args.files.iter(); // maybe `.peekable()` too if we want to check if the iterator is empty
// ..
if let Some((first, _)) = files.next() {
    // ..
}
```

Reply:
I agree, just tried to keep refactoring to a minimum.